templates/zerver/register.html

```diff
@@ -155,22 +147,16 @@
           <div class="help-box">
             <div id="org_type_blob_1" class="blob display-none">
               {% trans %}
-              Create a corporate organization if your users will be members of the
-              same company, where the privacy expectation is that the company's
-              organization administrators may need to have access to your users'
-              private message history. Zulip features that allow organization
-              administrators to take control of accounts or access private message
-              history for other users are only available to corporate organizations.
+              Organization admins will be able to take control of users'
```

Comment:
I would just move the commit that removes the org type entirely to come before this commit.